crates/nu-command/tests/commands/url/parse.rs

```diff
@@ -0,0 +1,172 @@
+use nu_test_support::{nu, pipeline};
+
+#[test]
+fn url_parse_simple() {
+    let actual = nu!(
+        cwd: ".", pipeline(
+            r#"
+                "https://www.abc.com"
+                | url parse
+            "#
+    ));
+
+    assert_eq!(
+        actual.out,
+        "╭──────────┬───────────────────╮\
+         │ scheme   │ https             │\
+         │ username │                   │\
+         │ password │                   │\
+         │ host     │ www.abc.com       │\
+         │ port     │                   │\
+         │ path     │ /                 │\
+         │ query    │                   │\
+         │ fragment │                   │\
+         │ params   │ {record 0 fields} │\
+         ╰──────────┴───────────────────╯"
```

Comment:
I think that we generally want to avoid testing the formatted output from `table` like this. Tests like this are fragile and will break whenever we tweak Nu's display logic.

Alternative approaches:
1. Figure out a way to test equality against a `Value::Record` (instead of text output) in Rust
2. Do the equality check against a Nu record in Nu code. Something like:

```rust
let actual = nu!(
    cwd: ".", pipeline(
        r#"
            ("https://www.abc.com" | url parse) == {scheme: 'https', username: '', ...}
        "#
));
assert_eq!(actual.out, "true");
```

Reply:
that makes sense, I just copied how some other tests were done. I will try to change this

Reply:
tests updated